src/pip/_internal/resolution/resolvelib/candidates.py

```diff
@@ -458,9 +467,14 @@ def format_for_error(self):
 
     @property
     def is_installed(self):
-        # type: () -> _BaseVersion
+        # type: () -> bool
```

Comment:
Just realised this was annotated incorrectly! Not sure why Mypy never reported this.

Reply:
Hmm... Do you wanna fix this now, or in a follow up PR? :)